test/suites/quickstatements/docker-compose.override.yml

```diff
@@ -2,38 +2,3 @@ services:
   wikibase:
     volumes:
       - ./quickstatements/LocalSettings.php:/var/www/html/LocalSettings.d/LocalSettings.quickstatements.php
-      - ./repo-LocalSettings.php:/var/www/html/LocalSettings.wbs.php
-      - ./repo-extra-install.sh:/extra-install.sh
-
-  wikibase-client:
```

Comment:
This cleans up an unrelated confusion in the tests. The Quickstatements Test Suite ran interwikilinks tests which were actually already part of the RepoClient test Suite. Removing it here to not run it twice and also simplify the config.

Reply:
I'll have to look at that, but sounds like a great move to me 👍🏼